### PR TITLE
media-sound/qpaeq: split out qpaeq package

### PR DESCRIPTION
**Pull request by i-garrison**

### CODE REVIEW

File: media-sound/pulseaudio-daemon/pulseaudio-daemon-16.1-r7.ebuild [hunk 1]
```diff
@@ -248,6 +248,11 @@ src_configure() {
 src_install() {
 	meson_src_install
 
+	# qpaeq moved to media-sound/qpaeq
```
Comment (thesamesam) [inline media-sound/pulseaudio-daemon/pulseaudio-daemon-16.1-r7.ebuild:251]: We'll have to add a blocker on media-sound/qpaeq and vice-versa in media-sound/qpaeq on <.../pulseaudio-daemon-16.1-r7.

File: media-sound/qpaeq/qpaeq-16.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
```
Comment (thesamesam) [inline media-sound/qpaeq/qpaeq-16.1.ebuild]: 8

**Review by thesamesam — COMMENTED** [review]
Thanks!

File: media-sound/qpaeq/qpaeq-16.1.ebuild (hunk 2 above)
Comment (i-garrison) [reply media-sound/qpaeq/qpaeq-16.1.ebuild]: amended this

File: media-sound/pulseaudio-daemon/pulseaudio-daemon-16.1-r7.ebuild (hunk 1 above)
Comment (i-garrison) [reply media-sound/pulseaudio-daemon/pulseaudio-daemon-16.1-r7.ebuild:251]: New `media-sound/qpaeq` dependends on `>=media-sound/pulseaudio-daemon-16.1-r7[dbus,fftw]`
I do not think that extra blocker is required, as qpaeq is via `optfeature`, correct?

Comment (i-garrison) [reply media-sound/pulseaudio-daemon/pulseaudio-daemon-16.1-r7.ebuild:251]: You are correct, added the blocker now.

**Pull request closed without merging**